DevOps/Communities/Readme.md

```diff
@@ -8,4 +8,5 @@
 - [Kubernetes Twitter](https://twitter.com/i/communities/1444745802383953921)
 - [Kubernetes Slack](https://kubernetes.slack.com/)
 - [CNCF Slack](https://slack.cncf.io/)
-- [DoK Slack](https://go.dok.community/slack)
\ No newline at end of file
+- [DoK Slack](https://go.dok.community/slack)
+- [kubesimplify](https://kubesimplify.com/discord)
```

Comment:
Please remove the upper website mention for Kubesimplify.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix